api/specs/storage/openapi.yaml

```diff
@@ -683,6 +691,27 @@ paths:
           title: Sha256 Checksum
         name: sha256_checksum
         in: query
+      - description: Page size limit
+        required: false
+        schema:
+          type: integer
+          maximum: 100
+          minimum: 1
+          title: Limit
+          description: Page size limit
+          default: 50
```

Comment:
I would even go lower here, say to 20, as we saw the large overhead this has. Just an idea...

Reply:
For starters I would like to have the same defaults in all paginations. We can ooptimize it later

Reply:
IN any case, if you look in detail, I am explicitly setting the `limit=1` in the calls we are optimizing here